### PR TITLE
Escape translated strings

### PR DESCRIPTION
**Pull request by Spreeuw**
Translations are currently barely escaped in our plugin. Traditionally we had full control over the source of our translations, but with a growing number of translation contributors on wordpress.org that can contribute translations without our approval, this is a potential attack vector. A malicious actor could (theoretically) use translations to create an XSS backdoor. The translator community is quite strong and I think the risk of this being effectively exploited is very small, but it should be `0`.

This PR addresses all translated strings before they are printed. PDF output (which does not have JS capabilities by default so is not a risk, theoretically) will be handled in a separate cleanup leg.

### CODE REVIEW

File: includes/class-wcpdf-admin.php [hunk 1]
```diff
@@ -129,8 +129,8 @@ public function install_wizard_notice() {
...
-					<p class="submit"><a href="<?php echo esc_url( admin_url( 'admin.php?page=wpo-wcpdf-setup' ) ); ?>" class="button-primary"><?php _e( 'Run the Setup Wizard', 'woocommerce-pdf-invoices-packing-slips' ); ?></a> <a href="<?php echo esc_url( add_query_arg( 'wpo_wcpdf_dismis_install', true ) ); ?>" class="wpo-wcpdf-dismiss-wizard"><?php _e( 'I am the wizard', 'woocommerce-pdf-invoices-packing-slips' ); ?></a></p>
+					<p><strong><?php esc_html_e( 'New to WooCommerce PDF Invoices & Packing Slips?', 'woocommerce-pdf-invoices-packing-slips' ); ?></strong> &#8211; <?php esc_html_e( 'Jumpstart the plugin by following our wizard!', 'woocommerce-pdf-invoices-packing-slips' ); ?></p>
+					<p class="submit"><a href="<?php echo esc_url( admin_url( 'admin.php?page=wpo-wcpdf-setup' ) ); ?>" class="button-primary"><?php esc_html_e( 'Run the Setup Wizard', 'woocommerce-pdf-invoices-packing-slips' ); ?></a> <a href="<?php echo esc_url( add_query_arg( 'wpo_wcpdf_dismis_install', true ) ); ?>" class="wpo-wcpdf-dismiss-wizard"><?php _e( 'I am the wizard', 'woocommerce-pdf-invoices-packing-slips' ); ?></a></p>
```
Comment (alexmigf) [inline includes/class-wcpdf-admin.php:133]: The `I am the Wizard` doesn't need escape?

File: includes/class-wcpdf-admin.php [hunk 2]
```diff
@@ -320,7 +331,7 @@ public function send_order_email_meta_box( $post ) {
 				<input type="submit" class="button save_order button-primary" name="save" value="<?php esc_attr_e( 'Save order & send email', 'woocommerce-pdf-invoices-packing-slips' ); ?>" />
 				<?php
 				$title = __( 'Send email', 'woocommerce-pdf-invoices-packing-slips' );
```
Comment (alexmigf) [inline includes/class-wcpdf-admin.php:333]: Maybe escape here too?

File: includes/class-wcpdf-admin.php (hunk 1 above)
Comment (Spreeuw) [reply includes/class-wcpdf-admin.php]: f1ecc51

File: includes/class-wcpdf-admin.php (hunk 2 above)
Comment (Spreeuw) [reply includes/class-wcpdf-admin.php:333]: this is not printed

File: includes/class-wcpdf-admin.php [hunk 3]
```diff
@@ -444,44 +466,44 @@ public function output_number_date_edit_fields( $document, $data ) {
...
 
 				<!-- Editable -->
 				<div class="editable">
 					<?php if( isset( $data['number'] ) ) : ?>
-					<p class="form-field <?= $data['number']['name']; ?>_field ">
-						<label for="<?= $data['number']['name']; ?>"><?= $data['number']['label']; ?></label>
-						<input type="text" class="short" style="" name="<?= $data['number']['name']; ?>" id="<?= $data['number']['name']; ?>" value="<?= $data['number']['plain']; ?>" disabled="disabled" > (<?= __( 'unformatted!', 'woocommerce-pdf-invoices-packing-slips' ) ?>)
+					<p class="form-field <?= esc_attr( $data['number']['name'] ); ?>_field">	
+						<label for="<?= esc_attr( $data['number']['name'] ); ?>"><?= wp_kses_post( $data['number']['label'] ); ?></label>
+						<input type="text" class="short" style="" name="<?= esc_attr( $data['number']['name'] ); ?>" id="<?= esc_attr( $data['number']['name'] ); ?>" value="<?= esc_attr( $data['number']['plain'] ); ?>" disabled="disabled" > (<?= esc_html__( 'unformatted!', 'woocommerce-pdf-invoices-packing-slips' ) ?>)
```
Comment (alexmigf) [inline includes/class-wcpdf-admin.php:500]: Missing ending semicolon on `esc_html__`

**Comment by alexmigf** [comment]
Fix indentation

```
			'defaultFont'				=> 'dejavu sans',

```
								
[includes/class-wcpdf-main.php](https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/aa08623aa62c9fddfbc3cdd10a5f40a5ea66e0fd/includes/class-wcpdf-main.php#L693) (Line 693)

**Comment by alexmigf** [comment]
Fix indentation

```
			'cache'			=> 'dompdf_font_family_cache.php',

```
								
[includes/class-wcpdf-main.php](https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/aa08623aa62c9fddfbc3cdd10a5f40a5ea66e0fd/includes/class-wcpdf-main.php#L706) (Line 706)

File: includes/class-wcpdf-settings-callbacks.php [hunk 4]
```diff
@@ -290,38 +288,38 @@ public function radio_button( $args ) {
...
 			}
 
-			$description = isset( $field['description'] ) ? '<span style="font-style:italic;">'.$field['description'].'</span>' : '';
+			$field_description = ! empty( $field['description'] ) ? '<span style="font-style:italic;">'.$field['description'].'</span>' : '';
 
 			// output field label
-			if (isset($field['label'])) {
-				printf( '<label for="%1$s_%2$s" %3$s>%4$s:</label>', $id, $name, $style, $field['label'] );
+			if ( isset( $field['label'] ) ) {
+				printf( '<label for="%1$s_%2$s" %3$s>%4$s:</label>', esc_attr( $id ), esc_attr( $name ), esc_attr( $style ), esc_attr( $field['label'] ) );
```
Comment (alexmigf) [inline includes/class-wcpdf-settings-callbacks.php:310]: Here the `$field['label']`is escaped with `esc_attr`, but other labels are with `esc_html`.

Comment (Spreeuw) [reply includes/class-wcpdf-settings-callbacks.php:310]: technically they work the same, but good catch - we should apply them consistently

**Comment by alexmigf** [comment]
Maybe escape this? In case the template gets hacked?

```
		echo $css;

```
								
[includes/documents/abstract-wcpdf-order-document.php](https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/720a7d428009560fbbe83a010e3d42bd37724d0d/includes/documents/abstract-wcpdf-order-document.php#L566) (Line 566)

**Comment by alexmigf** [comment]
Should we escape this function and below ones?

```
		echo $this->get_shop_name();

```
								
[includes/documents/abstract-wcpdf-order-document.php](https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/720a7d428009560fbbe83a010e3d42bd37724d0d/includes/documents/abstract-wcpdf-order-document.php#L660) (Line 660)

**Comment by alexmigf** [comment]
Escape here?

```
				<p><?php printf( __( 'The PDF files in %s are not currently protected due to your site running on <strong>NGINX</strong>.', 'woocommerce-pdf-invoices-packing-slips' ), '<strong>' . $tmp_path . '</strong>' ); ?></p>

```
								
[woocommerce-pdf-invoices-packingslips.php](https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/720a7d428009560fbbe83a010e3d42bd37724d0d/woocommerce-pdf-invoices-packingslips.php#L371) (Line 371)

**Review by alexmigf — APPROVED** [review]

**Pull request merged**